packages/utils/src/misc.ts

```diff
@@ -31,7 +31,13 @@ export function uuid4(): string {
       return crypto.randomUUID().replace(/-/g, '');
     }
     if (crypto && crypto.getRandomValues) {
-      getRandomByte = () => crypto.getRandomValues(new Uint8Array(1))[0];
+      getRandomByte = () => {
+        // crypto.getRandomValues might return undefined instead of the typed array
+        // in old Chromium versions (e.g. 23.0.1235.0 (151422))
+        const typedArray = new Uint8Array(1);
```

Comment:
Let's add why this change works:

(feel free to adjust my suggestion if you prefer)

```suggestion
        // crypto.getRandomValues might return undefined instead of the typed array
        // in old Chromium versions (e.g. 23.0.1235.0 (151422))
        // However, `typedArray` is still filled in-place.
        const typedArray = new Uint8Array(1);
```

Reply:
Sure! I just included your comment lines